go.mod

```diff
@@ -1,31 +1,33 @@
-module github.com/github.com/conduitio-labs/conduit-connector-notion
+module github.com/conduitio-labs/conduit-connector-notion
 
 go 1.19
 
-require github.com/conduitio/conduit-connector-sdk v0.3.0
+require (
+	github.com/conduitio/conduit-connector-sdk v0.2.0
```

Comment:
Umm, I this won't work with the latest version of Conduit, is this on purpose?

Reply:
Yes - at the moment the platform is running v0.2.1. We need this connector (or even, the initial version of this connector) to be compatible with the platform, until we can prep the platform to upgrade to latest Conduit